packages/react-code-editor/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -551,6 +569,39 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
                     <CodeEditorContext.Provider value={{ code: value }}>{customControls}</CodeEditorContext.Provider>
                   )}
                 </div>
+              }
+              {<div className={css(styles.codeEditorHeaderMain)}>{headerMainContent}</div>}
+              {!!shortcuts.length && (
+                <div className="pf-c-code-editor__keyboard-shortcuts">
+                  <Popover
+                    aria-label="Shortcuts"
+                    bodyContent={
+                      <Grid span={6} hasGutter>
```

Comment:
I wonder how much customization we want to expose for this popover. Should we allow users to customize popover props like direction, or the content of the body? What do others think?

There's also some additional white space created compared to the design because of Grid breaking on the word when it hits the width, but I'm not sure there's a good way to handle that beyond setting a larger popup width or adjust the grid gap css.